cloudsim-plus/src/main/java/org/cloudbus/cloudsim/resources/File.java

```diff
@@ -83,11 +89,18 @@ public File(final String fileName, final int fileSize) {
     public File(final File file) throws IllegalArgumentException {
         Objects.requireNonNull(file);
 
-        init(file.getName(), file.getSize());
+        if (file.getSize() <= 0) {
```

Comment:
One constructor must call another one to avoid code duplication and bugs. This constructor must call the first one.